docs/en/ingest-management/security/certificates.asciidoc

```diff
@@ -244,6 +252,17 @@ to the other {agents}
 Private key to use to expose this {fleet-server} HTTPS endpoint 
 to the other {agents}
 
+`elastic-agent-cert`::
+The certificate to use as the client certificate for {agent}'s connections to {fleet-server}.
+`elastic-agent-cert-key`::
+The path to the private key to use as for {agent}'s connections to {fleet-server}.
+`fleet-server-es-cert`::
+The path to the client certificate that {fleet-server} will use when connecting to {es}.
+`fleet-server-es-cert-key`::
+The path to the private key that {fleet-server} will use when connecting to {es}.
+`fleet-server-client-auth`::
+One of `none`, `optional`, or `required`. Defaults to `none`. {fleet-server}'s client_authentication option for client mTLS connections. If `optional` or `required` is specified, client certificates are verified using CAs specified in the `--certificate-authorities` flag.
```

Comment:
Consider front-loading the description

```suggestion
{fleet-server}'s client_authentication option for client mTLS connections. Accepted values: one of `none`, `optional`, or `required`. Defaults to `none`. If `optional` or `required` is specified, client certificates are verified using CAs specified in the `--certificate-authorities` flag. 
```

Reply:
Sorry @shainaraskas! I pulled the trigger too early and didn't notice this.